engine/src/cython/initialize.cpp

```diff
@@ -151,5 +153,11 @@ void blazingSetAllocator(
 	for (size_t i = 0; i < devices.size(); ++i)
 		rmmValues.devices.push_back(devices[i]);
 
-	BlazingRMMInitialize(&rmmValues);
+	float BLAZING_DEVICE_MEM_RESOURCE_CONSUMPTION_THRESHOLD = 0.95;
```

Comment:
use normal naming convention for variables.

Reply:
Constants are traditionally set as ALL CAPS. We have used that pattern for all configurable parameters in the past. This is sticking with our current naming convention for config parameters which as has been to set them as all caps.